Composer/packages/client/src/components/ProjectTree/ProjectTree.tsx

```diff
@@ -167,6 +167,13 @@ export const ProjectTree: React.FC<Props> = ({
     return process.env.COMPOSER_ENABLE_FORMS && dialog.content?.schema !== undefined;
   };
 
+  const formDialogSchemaExists = (projectId: string, dialog: DialogInfo) => {
+    return (
+      dialogIsFormDialog(dialog) &&
```

Comment:
I coded dialogIsFormDialog to return false if the feature switch is false.  If the user deletes the dialog with the feature switch off, will it leave files on disk?

Reply:
If the feature flag is not set, user should not see the menu item.